src/ASM/ASMs2Dmx.C

```diff
@@ -1009,3 +1009,13 @@ void ASMs2Dmx::generateThreadGroups (const Integrand& integrand, bool silence)
 
   ASMs2D::generateThreadGroups(p1-1, p2-1, silence);
 }
+
+
+void ASMs2Dmx::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
+{
+  if (basis > 0)
+    this->ASMs2D::getBoundaryNodes(lIndex, nodes, basis);
+  else
+    for (size_t b = 1; b <= this->getNoBasis(); ++b)
+      ASMs2D::getBoundaryNodes(lIndex, nodes, b);
+}
```

Comment:
Inkonsekvent med this-> på den ene og ikke den andre (er vel samme funksjon som kalles). Foretrekker på bruke this-> på alle member-metode kall, så det er tydelig at det er en member funksjon og ikke en global (lesbarhet).
Gjelder flere steder i denne commiten